db/seeds/main.go

```diff
@@ -213,6 +265,54 @@ func createStores() {
 	}
 }
 
+func createOrders() {
+	var users []models.User
+	if err := db.DB.Limit(480).Preload("Addresses").Find(&users).Error; err != nil {
+		log.Fatalf("query users (%v) failure, got err %v", users, err)
+	}
+
+	var sizeVariations []models.SizeVariation
+	if err := db.DB.Find(&sizeVariations).Error; err != nil {
+		log.Fatalf("query sizeVariations (%v) failure, got err %v", sizeVariations, err)
+	}
+	var sizeVariationsCount = len(sizeVariations)
+
+	for i, user := range users {
+		user.CreatedAt = randTime()
```

Comment:
> Why change CreatedAt here? but not when create users?

Because set `CreatedAt` when create a new user doesn't working. And I want to made sure the `order.CreatedAt` is after `user.CreatedAt`.